packages/aws-appsync-subscription-link/src/subscription-handshake-link.ts

```diff
@@ -51,7 +51,7 @@ export class SubscriptionHandshakeLink extends ApolloLink {
         this.subsInfoContextKey = subsInfoContextKey;
     }
 
-    request(operation: Operation) {
+    request(operation: Operation): Observable<FetchResult> | null {
```

Comment:
hmm I dont if the typing is right here. [Looking at this line](https://github.com/awslabs/aws-mobile-appsync-sdk-js/blob/213f0cf4e807d91810153e8048be863f630ca3e1/packages/aws-appsync-subscription-link/src/subscription-handshake-link.ts#L72), it might be observable of a different type too.

Would you be interested in taking a closer look at this?

Reply:
The linked line appears to be type `Observable<unknown>` since there is no `next` on the observable. I think we could cast the observable on line 71 to be `Observable<FetchResult>`.